src/System.Management.Automation/help/ProviderContext.cs

```diff
@@ -110,14 +110,11 @@ internal MamlCommandHelpInfo GetProviderSpecificHelpInfo(string helpItemName)
             // Does the provider know how to generate MAML.
             CmdletProvider cmdletProvider = providerInfo.CreateInstance();
             ICmdletProviderSupportsHelp provider = cmdletProvider as ICmdletProviderSupportsHelp;
-            if (provider == null)
-            {
-                return null;
-            }
 
-            if (resolvedProviderPath == null)
+            // Under JEA sessions the resolvedProviderPath will be null, we should allow get-help to continue.
+            if (provider == null || resolvedProviderPath == null)
```

Comment:
Strange but my previous comment no longer appears.  Anyway my concern with this change is that previously an exception was thrown if the resolvedProviderPath is null, but now it continues.  This feels like a breaking change since it now can return incomplete help for non-JEA sessions.  I am not sure if that is a big deal but maybe it would be possible to detect JEA session before skipping the error.